src/components/TemplateCard/TemplateCard.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react'
+import Template from '../../ducks/Studio/Template/Dialog/LoadTemplate/Template'
+import styles from './TemplateCard.module.scss'
+
+const TemplateCard = ({ template, isAuthor }) => {
+  return (
+    <div className={styles.card}>
+      <Template template={template} isAuthor={isAuthor} />
+    </div>
+  )
+}
```

Comment:
Looks like this component need for only new wrapper styles. 
Maybe we can just pass `className` to `Template` directly and skip this component?